### DISCUSSION TITLE
v0.1.4 - Easy Launcher

### DISCUSSION THREAD

**Discussion by CreativeCodeCat** [DroidWorksStudio/EasyLauncher#67] [Check for Updates]
# What's New

### Implemented Enhancements:

- Added auto update. - ([2dcb4ae](https://github.com/DroidWorksStudio/EasyLauncher/commit/2dcb4ae4b8dacb229ae4664470cbb3b08d8348b4))
- Fixed settings location to click. - ([c5e9117](https://github.com/DroidWorksStudio/EasyLauncher/commit/c5e91179deedceee9be795f81552a5d141448ec5))

### Housekeeping:

- Removed unneeded imports - ([5066d2b](https://github.com/DroidWorksStudio/EasyLauncher/commit/5066d2b3598e2ff0a4db9a3618dd157134659839))

### Language Support:

- Updated Language Files. ([#66](https://github.com/DroidWorksStudio/EasyLauncher/issues/66)) - ([3b95995](https://github.com/DroidWorksStudio/EasyLauncher/commit/3b959953e0e6fa54e1a4d6db7a63fa8afddbe413))
- Updated Language Files. ([#62](https://github.com/DroidWorksStudio/EasyLauncher/issues/62)) - ([803f47f](https://github.com/DroidWorksStudio/EasyLauncher/commit/803f47f6b0e8408e79f73bb3c02ce434a42e3955))

### Refactoring:

- Refactored the settings for widgets. - ([91d34ae](https://github.com/DroidWorksStudio/EasyLauncher/commit/91d34ae6b37a9efc51eda98d6d0351b2c9ce6a50))

### Releases:

- Prepare for version 0.1.4 - ([db9a390](https://github.com/DroidWorksStudio/EasyLauncher/commit/db9a39000ce1200b39738a5bd75853989a35c5de))

**Full Changelog**: https://github.com/DroidWorksStudio/EasyLauncher/compare/0.1.3...0.1.4

<hr /><em>This discussion was created from the release <a href='https://github.com/DroidWorksStudio/EasyLauncher/releases/tag/0.1.4'>v0.1.4 - Easy Launcher</a>.</em>

**Comment by Akselmo** [comment]
Hi, will the widget list allow custom widgets too eventually? I would love to add my calendar and orgmode-notes-app widgets there. :)

**Reply by CreativeCodeCat** [reply]
Not as for now as allowing custom widgets I have been having issues with maybe one day but not for now.

**Reply by Akselmo** [reply]
Understandable! Nothing wrong with keeping scope small.
